2024/door_19/README.md

```
@@ -0,0 +1,19 @@
+# Getting Started
+Install the dependencies and run the project
+```
+npm install
+npm start
+```
+
+Head over to https://vitejs.dev/ to learn more about configuring vite
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)